GeoLite2 database not free anymore

<!--
Before opening an issue, just take into account that this is a completely free of charge open source project.
I'm always happy to help and provide support, but some understanding will be required.
I do this in my own free time, so expect some delays when implementing new features and fixing bugs, and don't take it personal if an issue gets eventually closed.
Try to be polite, and understand it is impossible for an OSS project to cover all use cases.

With that said, please fill in the information requested next. More information might be requested next (like logs or system configs).
-->

#### How Shlink is set-up

* Shlink Version: 1.21.0
* PHP Version: 7.3.13
* How do you serve Shlink: Self-hosted swoole
* Database engine used: MariaDB

#### Summary

The URL `http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz` doesn't exist anymore. 

More here: https://dev.maxmind.com/geoip/geoip2/geolite2/
Maybe including there package? https://maxmind.github.io/GeoIP2-php/
<!-- Provide a summary describing the problem you are experiencing. -->

#### Current behavior
`bin/cli visit:update-db` results in `[ERROR] An error occurred while updating GeoLite2 database`

#### Expected behavior

<!-- How did you expected to behave? -->

#### How to reproduce

<!-- Provide steps to reproduce the bug. -->


If i'm incorrect please let me know but upgrading doesn't work for me right now. Very annoying

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## 1.21.1 - 2020-01-02
+
+#### Added
+
+* *Nothing*
+
+#### Changed
+
+* *Nothing*
+
+#### Deprecated
+
+* *Nothing*
+
+#### Removed
+
+* *Nothing*
+
+#### Fixed
+
+* [#596](https://github.com/shlinkio/shlink/issues/596) Fixed error when trying to download GeoLite2 database due to changes on how to get the database files.
+
+
 ## 1.21.0 - 2019-12-29
 
 #### Added

--- a/config/autoload/geolite2.global.php
+++ b/config/autoload/geolite2.global.php
@@ -7,7 +7,9 @@ return [
     'geolite2' => [
         'db_location' => __DIR__ . '/../../data/GeoLite2-City.mmdb',
         'temp_dir' => sys_get_temp_dir(),
-        'download_from' => 'http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz',
+        'download_from' =>
+            'https://download.maxmind.com/app/geoip_download'
+            . '?edition_id=GeoLite2-City&license_key=G4Lm0C60yJsnkdPi&suffix=tar.gz',
     ],
 
 ];

